StreamChatUITestsAppUITests/Tests/Performance/LoadTime.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+
+@available(iOS 15.0, *)
+class LoadTime: StreamTestCase {
+    
+    func testMessageList() {
+        WHEN("user logs in") {
+            backendRobot.generateChannels(count: 1, messagesCount: 200)
+            userRobot.login()
+        }
+        THEN("user opens the channel") {
+            let measureOptions = XCTMeasureOptions()
+            measureOptions.invocationOptions = [.manuallyStop]
+            measure(metrics: [XCTClockMetric()], options: measureOptions) {
+                userRobot.openChannel()
+                MessageListPage.cells.waitCount(1)
+                stopMeasuring()
+                userRobot.tapOnBackButton().logout().login()
+            }
+        }
+    }
+    
+    func testChannelList() {
+        WHEN("channels are generated") {
+            backendRobot.generateChannels(count: 1, messagesCount: 200)
```

Comment:
1 channel is not enough, we should test with the default of 25. Parsing larger channels is one of the things we should track. Messages count can also be 25, doesn't need to be 200 for the channel list.

Reply:
Nice catch, copy-pasted from `testMessageList` and forgot to update.

Reply:
Updated 👍